CMakeLists.txt

```diff
@@ -2,6 +2,11 @@ cmake_minimum_required(VERSION 3.5)
 
 project(open_simulation_interface)
 
+# set default apple compiler
```

Comment:
I think this should very likely be done unconditionally anyway, since e.g. the OSMP examples already turn this on by themselves. If someone needs to compile OSI against non-C++11 compilers, they are likely to change this as needed.

Reply:
@pmai rework request processed :-)